ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -198,7 +199,7 @@
     <i>inline-block</i>
     to
     <i>block</i>, set
-    <kbd class="ds-kbd">@isInlineBlock</kbd>
+    <code class="ds-code">@isInlineBlock</code>
```

Comment:
I don't know if we can use `<code>` by itself (I think it allows us to, but I don't know if we _should_) so let me check this first; otherwise seems good.